common/src/main/java/com/trainlab/service/impl/UserServiceImpl.java

```diff
@@ -60,8 +59,7 @@ private void setEncodedPassword(User user) {
 
     @Override
     public void activateUser(String userEmail) {
-        User user = userRepository.findByAuthenticationInfoEmail(userEmail)
-                .orElseThrow(() -> new EntityNotFoundException("User with email not found"));
+        User user = findByEmail(userEmail);
 
         if (!user.isActive()) {
             user.setActive(true);
```

Comment:
Удобно писать так чтобы было меньше if else конструкций
К примеру тут мы можем сделать 
елси юзер активен, тогда кидаем эксепшн
иначе выполняем логику, но можно опустить блок else и просто написать код ниже, это и будет подразумевать, что если код дошел до этого момента, значит не выбросило эксепшн, значит юзер не активный

```suggestion
         if (user.isActive()) {
          throw new IllegalStateException("User with email " + userEmail + " is yet activate.");
        }
        
          user.setActive(true);
          user.setChanged(Timestamp.valueOf(LocalDateTime.now()));
          userRepository.saveAndFlush(user);
          log.info("User with email " + userEmail + " activate successfully!");
```